Winapp2.ini

```diff
@@ -2709,6 +2710,14 @@ FileKey17=%Documents%\Adobe\Adobe Media Encoder\*\logs|*.*|RECURSE
 FileKey18=%Documents%\Adobe\Premiere Pro\*|Plugin Loading.log
 RegKey1=HKCU\Software\Adobe\MediaBrowser\MRU
 
+[Adobe Creative Cloud Experience *]
+LangSecRef=3023
+DetectFile=%ProgramFiles%\Adobe\Adobe Creative Cloud Experience
+Default=False
+FileKey1=%ProgramFiles%\Adobe\Adobe Creative Cloud Experience\js\node_modules\table-parser\test\output|*.log
+FileKey2=%ProgramFiles%\Common Files\Adobe\Creative Cloud Libraries\js\node_modules\table-parser\test\output|*.log
+FileKey3=%ProgramFiles%\Common Files\Adobe\Creative Cloud Libraries\js\node_modules\restify|*.log
+

```

Comment:
Please use for "%ProgramFiles%\Common Files\" the variable %CommonProgramFiles%.